src/librustc_mir/borrow_check/nll/region_infer/error_reporting/mod.rs

```diff
@@ -540,14 +541,15 @@ impl<'tcx> RegionInferenceContext<'tcx> {
                             RegionName::Named(name) => format!("{}", name),
                             RegionName::Synthesized(_) => "'_".to_string(),
                         };
-                        diag.span_suggestion(
+                        diag.span_suggestion_with_applicability(
                             span,
                             &format!(
                                 "to allow this impl Trait to capture borrowed data with lifetime \
                                  `{}`, add `{}` as a constraint",
                                 fr_name, suggestable_fr_name,
                             ),
                             format!("{} + {}", snippet, suggestable_fr_name),
+                            Applicability::MachineApplicable,
```

Comment:
I feel it might be safer to use `MaybeIncorrect` here, but this might be fine in the wild. Lets keep it this way and fix it if we find positive proof that it is a problem.